### ISSUE TITLE
retinanet accuracy

### ISSUE THREAD

**Issue by meixitu** [hasanirtiza/Pedestron#18]
Hi @hasanirtiza ,
    In your pretrained model of Retinanet, the cityperson accuracy is not that good, did you use crowdHuman and widerpedestrian dataset to pre-trained it?

Thanks
Meixitu

**Comment by hasanirtiza** [comment]
Yes, the model we published for RetinaNet is pre-trained on CrowdHuman. You can read more about it in our [pre-print version](https://arxiv.org/pdf/2003.08799.pdf). Furthermore, #3.

**Comment by meixitu** [comment]
@hasanirtiza 

Thanks for your reply.
So I think the conclusion is RetinaNet is not powerful enough.
for face detection, retinanet can get very good accuracy, I don't know why it is not good in pedestrian detection. 

Thanks
Meixitu

**Comment by hasanirtiza** [comment]
We have some better numbers with the RetinaNet for pedestrian detection. We plan to release the model in future hopefully soon. Of course do not expect it to be up there with Cascade etc.

**Comment by hasanirtiza** [comment]
One more comment, for the face detection, multi scale testing plays a very pivotal role for better accuracy.

**Comment by meixitu** [comment]
for your current published score, do you just use the original image size?

**Comment by hasanirtiza** [comment]
Yes, most if not all models are evaluated using the original image size.

**Comment by meixitu** [comment]
Thank you very much

Thanks
Meixitu

**Comment by meixitu** [comment]
@hasanirtiza 

    You said you can get better performance for retinaNet, what did you modified? Could you give me some clue?

Thanks
Meixitu

**Comment by hasanirtiza** [comment]
We will share it soon (keep checking the repo) , since we are still in the process training, as of now unfortunately we cannot share more details.

**Comment by hasanirtiza** [comment]
@meixitu Checkout the README.md. A new model of RetinaNet has been added.

**Comment by meixitu** [comment]
@hasanirtiza ,

Got it.

Thanks
Meixitu
